bucket/prusaslicer.json

```
@@ -24,17 +19,13 @@
     ],
     "checkver": {
         "url": "https://api.github.com/repos/prusa3d/PrusaSlicer/releases",
-        "regex": "(?sm)PrusaSlicer-[\\d.]+\\+win32-(?<timestamp32>\\d+)\\.zip.*?PrusaSlicer-([\\w.-]+)\\+win64-(?<timestamp64>\\d+)\\.zip"
+        "regex": "PrusaSlicer-([\\w.-]+)\\+win64-(?<timestamp64>\\d+)\\_signed.zip"
```

```suggestion
        "regex": "PrusaSlicer-([\\w.-]+)\\+win64-(?<timestamp64>\\d+)_signed\\.zip"
```

The regex escape `\_` is invalid for the .NET variant.
https://github.com/ScoopInstaller/Extras/runs/8270519843?check_suite_focus=true
```
Line | 
     290 |  …     $regex = New-Object System.Text.RegularExpressions.Regex($regexp) 
         |                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
         | Exception calling ".ctor" with "1" argument(s): "Invalid pattern 
         | 'PrusaSlicer-([\w.-]+)\+win64-(?<timestamp64>\d+)\_signed.zip' at offset 50. Unrecognized escape 
         | sequence \\_." 
```

---

Thank you very much. I will more about regular expressions